packages/std/src/lib.rs

```diff
@@ -55,10 +55,10 @@ pub use crate::results::{
     attr, wasm_execute, wasm_instantiate, Attribute, BankMsg, ContractResult, CosmosMsg, Empty,
     Event, QueryResponse, Reply, ReplyOn, Response, SubMsg, SubcallResponse, SystemResult, WasmMsg,
 };
-#[allow(deprecated)]
-pub use crate::results::{Context, HandleResponse, InitResponse, MigrateResponse};
 #[cfg(feature = "staking")]
 pub use crate::results::{DistributionMsg, StakingMsg};
+#[allow(deprecated)]
+pub use crate::results::{HandleResponse, InitResponse, MigrateResponse};
```

Comment:
Ahh, good to see this here. Those 3 should be removed too: https://github.com/CosmWasm/cosmwasm/issues/952. Could you close both tickets with this one PR?

Reply:
Sure!